transformer_engine/pytorch/transformer.py

```diff
@@ -77,6 +77,48 @@ def forward(self, hidden_state: torch.Tensor) -> torch.Tensor:
         output = hidden_state.div(keep_prob) * random_tensor
         return output
 
+class _SplitLastDim(torch.autograd.Function):
```

Comment:
This is to save a call to the concat kernel in the backward pass in case the tensor split during forward was contiguous (makes sense). However, has this code path been tested? I believe the 126m model in ci uses dimensions that would circumvent this call.

Reply:
The change passed internal training tests.